app/components/generic/charts/DoughnutChart.tsx

```diff
@@ -63,8 +72,8 @@ export default function DoughnutChart({
         },
         tooltip: {
           callbacks: {
-            label: function (context: TooltipItem<keyof ChartTypeRegistry>) {
-              const value = valueFormatter(Number(context.raw as number));
+            label(context: TooltipItem<'doughnut'>) {
+              const value = valueFormatter(context.parsed);
```

Comment:
nitpick: is it possible to return directly instead of declaring const value and then returning it?

```suggestion
              return valueFormatter(context.parsed);
```

Reply:
changed